test/wrap.spec.js

```diff
@@ -11,9 +11,10 @@ describe('wrap', () => {
     });
     class MyLily extends HTMLElement {}
 
-    html`<${MyLily} id="${'my-id'}">${'my text'}</${MyLily}>`;
+    html`<${MyLily} id="${'my-id'}" style=${null}>${'my text'}</${MyLily}>`;
+    console.warn(calledArgs);
     expect(calledCount).to.equal(1);
-    expect(calledArgs).to.deep.equal([['<my-lily id="', '">', '</my-lily>'], 'my-id', 'my text']);
+    expect(calledArgs).to.deep.equal([['<my-lily id="', '" style=', '>', '</my-lily>'], 'my-id', null, 'my text']);
```

Comment:
this logic belongs to "src/transform.js" and is better to test in "test/transform.spec.js"

I'd suggest to write a new test after this one, e.g. `it('does not throw on nullish value')`, and provide all sort of nullish values that you are interested in,  one test is OK

```
  it('registers names for Custom Element classes in values and concatenates strings with those names', () => {
    class MyTulip extends HTMLElement {}
    const [strings, ...values] = transform(...testhtml`<${MyTulip} id="${'my-id'}">${'my text'}</${MyTulip}>`);
    expect(strings).to.deep.equal(['<my-tulip id="', '">', '</my-tulip>']);
    expect(values).to.deep.equal(['my-id', 'my text']);
    expect(customElements.get('my-tulip')).to.equal(MyTulip);
  });
```

Reply:
Done, thanks for your patience 🌞